src/fetchData/type.ts

```diff
@@ -0,0 +1,8 @@
+import { Resources } from '../utils/response/apiResponse';
+import { RawApiResponse } from '../utils/response/rawApiResponse';
+
+export type FetchDataResponse = {
+  success: boolean;
+  apiResponse: RawApiResponse;
+  results: Partial<Resources>;
```

Comment:
not sure about this line. because it is `Partial<Resources>`. DeltaSync has a bit different definition. Each resource has data which has type `ResourceResponse` (for example `ResourceResponse<AbsenceDay>`)